snorkel/classification/models/__init__.py

```diff
@@ -1,2 +1 @@
 from .advanced import AdvancedClassifier, Operation, Task  # noqa: F401
```

Comment:
`AdvancedClassifier` is a bad name here- it doesn't mean anything, and even more so when there's no `SimpleClassifier`.  So I think this needs to be changed in this PR.  As for alternatives- `TaskFlowClassifier` also sounds a bit wonky to me.  I don't think "task flow" will mean anything to anyone other than a few people in the lab...

More broadly than just naming: I am still a little confused as to why we don't just have `SingleTaskClassifier` (perhaps shortened or aliased to `Classifier`) and `MultiTaskClassifier`?  This is a straightforward distinction that anyone will understand and seems cleaner